Adyen/Core/Core Protocols/PaymentMethod.swift

```diff
@@ -15,27 +15,39 @@ public protocol PaymentMethod: Decodable {
     /// The name of the payment method, such as `"Credit Card"`, `"iDEAL"`, `"Apple Pay"`.
     var name: String { get }
     
-    /// Display information for the payment method, adapted for displaying in a list.
-    var displayInformation: DisplayInformation { get }
+    /// Describes a payment method display information thats provided by the merchant
+    /// and if not `nil`, will override the display information generated by ``defaultDisplayInformation(using:)``.
+    var merchantProvidedDisplayInformation: MerchantCustomDisplayInformation? { get set }
     
     /// Display information for the payment method, adapted for displaying in a list.
     ///
     /// - Parameters:
     ///   - using: The localization parameters.
-    func localizedDisplayInformation(using parameters: LocalizationParameters?) -> DisplayInformation
+    /// :nodoc:
+    func defaultDisplayInformation(using parameters: LocalizationParameters?) -> DisplayInformation
     
     /// :nodoc:
     func buildComponent(using builder: PaymentComponentBuilder) -> PaymentComponent?
 }
 
 /// :nodoc:
 public extension PaymentMethod {
-
-    var displayInformation: DisplayInformation {
-        DisplayInformation(title: name, subtitle: nil, logoName: type.rawValue)
+    
+    func displayInformation(using parameters: LocalizationParameters?) -> DisplayInformation {
+        let defaultDisplayInformation = defaultDisplayInformation(using: parameters)
+        if let merchantProvidedDisplayInformation = merchantProvidedDisplayInformation {
+            let subtitle = merchantProvidedDisplayInformation.subtitle ?? defaultDisplayInformation.subtitle
```

Comment:
Is there are cases when subtitle is just informative, not payment specific (masked PAN, email e.t.c) ?
Maybe we should not allow to change `subtitle`, as we do with `logoName` ?

Reply:
I don't see why not allow `subtitle` to be overiden, its up to merchants